README.md

```diff
@@ -389,6 +389,61 @@ describe App do
 end
 ```
 
+In order to test the client side `stub_request` with Webmock.
```

Comment:
Building on the above, I think this part should say ...

In order to stub the response to actual queries, [dump the schema into a JSON file](#schema-storing-and-loading-on-disk) and specify it via `schema_path` as follows.

```ruby
# your example
```

Now the first note is taken care by the intro, and the second note is unrelated here - I think if we're going to describe what happens in camel case vs. underscore it should be its own section and we need to show where this happens and why.

Reply:
Auto camelCase <-> snake-case happens in [graphq-ruby](https://github.com/rmosolgo/graphql-ruby/blob/588c2fde44ca8cf4565a4a852fda2a251316b53b/guides/fields/arguments.md)